README.md

```diff
@@ -1 +1,153 @@
-# FIRST-CAPSTONE-PROJECT
\ No newline at end of file
+# HTML CSS and javascript capstone project
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    -[Tech Stack](#tech-stack)
+- [🚀 Live Demo](#live-demo) 
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+-- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [HTML CSS and javascript capstone project] <a name="HTML CSS and javascript capstone project"></a>
+
+
+**[HTML CSS and javascript capstone project]** is a HTML, CSS and JavaScript project that teaches how to Use JavaScript to manipulate DOM elements, Use JavaScript events, Use objects to store and access data, use Flexbox to place elements in the page, use media query to build a fully responsive personal portfolio site.
+
+## 🛠 Built With  <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <summary>Javascript runtime environment</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node JS</a></li>
+  </ul>
+
+  <summary>Version control</summary>
+  <ul>
+    <li><a href="github.com">Git Hub</a></li>
+  </ul>
+</details>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+Click here to see a live demo of this project online: <li><a href="https://abdelazizsoliman.github.io/FIRST-CAPSTONE-PROJECT//">Live Demo</a></li>
+
+Click here to see a video presentation on LOOM: <li><a href="https://www.loom.com/share/a0e0304dbc0740c5991d6196243f7e60">Show video</a></li>
```

Comment:
[REQUIRED] Good job creating a video presentation but you did not highlight a piece of code that you are pleased with.